USFMToolsSharp.Renderers.Docx/Extensions/XWPFParagraphExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NPOI.XWPF.UserModel;
+
+
+namespace USFMToolsSharp.Renderers.Docx.Extensions
+{
+    public static class XWPFParagraphExtensions
+    {
+        public static void StyleParagraph(this XWPFParagraph para, StyleConfig styles)
```

Comment:
You should actually create an extension method for CreateParagraph on document that does this instead of adding another function call.

Reply:
I reattached the extension methods to XWPFDocument, XWPFParagraph implementing CreateStyledParagraph. CreateStyledRun respectively.

Reply:
Should be named the same as the other in my opinion